.gitignore

```diff
@@ -15,3 +15,4 @@
 /sys/libyuv-sys/src/libyuv.rs
 /sys/ndk-sys/src/bindings.rs
 /target
+/third_party
```

Comment:
Commit `0dda3e7d5b25b29a0ee0c93b07d77e2243bf645b` renamed `third_party` to `external`.  Any reason why you still need to ignore third_party?